dist/index.js

```diff
@@ -0,0 +1,2 @@
ger(this._element,"hide.bs.toast").defaultPrevented||(this._element.classList.add(Dr),this._queueCallback((()=>{this._element.classList.add(Lr),this._element.classList.remove(Dr),this._element.classList.remove(Sr),pe.trigger(this._element,"hidden.bs.toast")}),this._element,this._config.animation)))}dispose(){this._clearTimeout(),this._element.classList.contains(Sr)&&this._element.classList.remove(Sr),super.dispose()}_getConfig(t){return t={...Ir,...ke.getDataAttributes(this._element),..."object"==typeof t&&t?t:{}},Bt(jr,t,this.constructor.DefaultType),t}_maybeScheduleHide(){this._config.autohide&&(this._hasMouseInteraction||this._hasKeyboardInteraction||(this._timeout=setTimeout((()=>{this.hide()}),this._config.delay)))}_onInteraction(t,e){switch(t.type){case"mouseover":case"mouseout":this._hasMouseInteraction=e;break;case"focusin":case"focusout":this._hasKeyboardInteraction=e}if(e)return void this._clearTimeout();const o=t.relatedTarget;this._element===o||this._element.contains(o)||this._maybeScheduleHide()}_setListeners(){pe.on(this._element,"mouseover.bs.toast",(t=>this._onInteraction(t,!0))),pe.on(this._element,"mouseout.bs.toast",(t=>this._onInteraction(t,!1))),pe.on(this._element,"focusin.bs.toast",(t=>this._onInteraction(t,!0))),pe.on(this._element,"focusout.bs.toast",(t=>this._onInteraction(t,!1)))}_clearTimeout(){clearTimeout(this._timeout),this._timeout=null}static jQueryInterface(t){return this.each((function(){const e=Mr.getOrCreateInstance(this,t);if("string"==typeof t){if(void 0===e[t])throw new TypeError(`No method named "${t}"`);e[t](this)}}))}}ue(Mr),Xt(Mr);var Pr=o(379),Br=o.n(Pr),Hr=o(795),Rr=o.n(Hr),$r=o(569),Wr=o.n($r),qr=o(565),Fr=o.n(qr),Ur=o(216),Vr=o.n(Ur),Xr=o(589),Yr=o.n(Xr),Kr=o(355),Qr={};Qr.styleTagTransform=Yr(),Qr.setAttributes=Fr(),Qr.insert=Wr().bind(null,"head"),Qr.domAPI=Rr(),Qr.insertStyleElement=Vr(),Br()(Kr.Z,Qr),Kr.Z&&Kr.Z.locals&&Kr.Z.locals;const Zr="https://us-central1-js-capstone-backend.cloudfunctions.net/api/",Gr="PemhzwzHYWKw2QgWQai4",Jr=async()=>{try{const t=await fetch(`${Zr}games/${Gr}/scores`);return await t.json()}catch(t){return t}},ti=t=>{const e=document.querySelector("#score");e.classList.remove("border"),e.innerHTML="",t.forEach((({user:t,score:o})=>{const r=document.createElement("li");r.textContent=`${t}: ${o}`,e.appendChild(r)})),e.classList.add("border")};(async()=>{const{result:t}=await Jr();ti(t)})(),document.querySelector("#refresh-btn").addEventListener("click",(async()=>{const{result:t}=await Jr();ti(t)})),document.querySelector("#add-a-score").addEventListener("submit",(async t=>{t.preventDefault(),t.stopPropagation();const e=document.querySelector("#name-input"),o=document.querySelector("#score-input");await(async t=>{try{const e=await fetch(`${Zr}games/${Gr}/scores`,{method:"POST",headers:{Accept:"application/json","Content-Type":"application/json"},body:JSON.stringify(t)});return await e.json()}catch(t){return t}})({user:e.value,score:o.value}),e.value="",o.value=""}))})()})();
```

Comment:
- [ ] This file has a standard name of `bundle.js` or `main.js`. The name you have used applies to the src directory.